.travis.yml

```diff
@@ -1,6 +1,16 @@
 language: rust
-rust: nightly
-script: cd test-project && cargo test
+rust:
+ - stable
+ - nightly
```

Comment:
You don't need these three lines:

```rust
rust:
 - stable
 - nightly
```

The `matrix` below does what you want already.

Reply:
Removed, thanks!